src/Core/Resolvers/BaseQueryStructure.cs

```diff
@@ -201,7 +200,7 @@ public SourceDefinition GetUnderlyingSourceDefinition()
         /// </summary>
         internal static IObjectField ExtractItemsSchemaField(IObjectField connectionSchemaField)
         {
-            return GraphQLUtils.UnderlyingGraphQLEntityType(connectionSchemaField.Type).Fields[QueryBuilder.PAGINATION_FIELD_NAME];
+            return connectionSchemaField.Type.NamedType<ObjectType>().Fields[QueryBuilder.PAGINATION_FIELD_NAME];
```

Comment:
I have refactored this to use the functionality from the core libs... asking for the `NamedType` will remove all GraphQL wrapper types.